about_arrays.go

```diff
@@ -14,6 +16,7 @@ func aboutArrays() {
 	assert(fruits == [4]string{}) // comparing arrays is not like comparing apples and oranges
 
 	tasty_fruits := fruits[1:3]           // defining oneself as a variation of another
+    assert(fmt.Sprintf("%T", tasty_fruits) == __string__) //and get not a simeple array as a result
```

Comment:
@dooman87 I think there is a spelling error here, `simeple` should be `simple` )

Reply:
Ah, thanks! Fixed it